otherlibs/systhreads/thread.ml

```diff
@@ -38,8 +38,9 @@ let create fn arg =
       try
         fn arg; ()
       with exn ->
-             flush stdout; flush stderr;
-             thread_uncaught_exception exn)
+        (try flush stdout with _ -> ()) ;
+        (try flush stderr with _ -> ()) ;
```

Comment:
Apologies for the naive question, but why are the changes to `Thread.create` part of this PR?

Reply:
Both have to do with possible raising exceptions while handling another (witness the name of the branch–I should have kept it as a title to be less misleading). So, in a separate commit, I included this minor and otherwise orphan change, hoping that we would not have to discuss it (c'est raté :).